lib/puppet/property/keyvalue.rb

```diff
@@ -12,9 +12,20 @@ class Property
     # @todo The node with an important message is not very clear.
     #
     class KeyValue < Property
```

Comment:
TODO: Double check that the KeyValue property preserves redacting of Sensitive values. (NOTE: Independent of this PR, this is just to make sure we don't have an underlying bug here).

Reply:
@branan Yup, I've verified using both the `keyvalue` array and the `hash` value that Sensitive values are correctly redacted. Here's a sample output:
```
[0] [AIX] root@aix71-5:~ # /opt/puppetlabs/puppet/bin/puppet apply MANIFESTS/pup_3021.pp
Notice: Compiled catalog for aix71-5.delivery.puppetlabs.net in environment production in 0.03 seconds
Notice: /Stage[main]/Main/User[USER_15]/attributes: changed [redacted] to [redacted]
Notice: Applied catalog in 0.35 seconds
```